packages/tasks/src/model-libraries.ts

```diff
@@ -130,6 +130,13 @@ export const MODEL_LIBRARIES_UI_ELEMENTS = {
 		filter: false,
 		countDownloads: `path:"asset/GPT.pt"`,
 	},
+	"colpali": {
+		prettyLabel: "ColPali",
+		repoName: "ColPali",
+		repoUrl: "https://github.com/ManuelFay/colpali",
+		filter: false,
+		countDownloads: `path:"adapter_model.safetensors"`,
```

Comment:
Maybe let's do `adapter_config.json` for consistency with other adapter libraries